namer/namer.cc

```diff
@@ -1414,6 +1414,57 @@ class SymbolDefiner {
         }
     }
 
+    void deleteViaFullNameHash(core::MutableContext ctx, const core::FoundFieldHash &oldDefHash) {
```

Comment:
very slight preference for `delete<Kind>ViaFullNameHash` instead of overloading. not blocking.